spec/features/saml/saml_spec.rb

```diff
@@ -481,6 +482,28 @@
 
         expect(current_url).to eq idv_url
       end
+
+      it 'redirects to SP after IdV is complete' do
```

Comment:
The expectation at the end of the this tests seems to be that we end up at the api/saml/auth endpoint, rather than the SP.. does the test need another `visit` or maybe the description needs updating?